crates/astria-composer/src/searcher/mod.rs

```diff
@@ -150,16 +154,29 @@ impl Searcher {
     /// + failed to connect to the eth RPC server;
     /// + failed to construct a sequencer clinet
     pub(super) async fn from_config(cfg: &Config) -> eyre::Result<Self> {
+        use rollup::Rollup;
+        let mut rollups = cfg
+            .rollups
+            .split(',')
+            .filter(|s| !s.is_empty())
+            .map(|s| Rollup::parse(s).map(Rollup::into_parts))
+            .collect::<Result<HashMap<_, _>, _>>()
+            .wrap_err("failed parsing provided <chain_id>::<url> pairs as rollups")?;
+        ensure!(
```

Comment:
We should have a test for this

Reply:
This is going to be removed in #278. This serves as a reminder.